ferrocene/doc/internal-procedures/src/manual-upstream-pulls.rst

```diff
@@ -10,97 +10,70 @@ What is an Upstream Pull
-script will exit. You'll have to fix the merge conflicts yourself in that case:
+In order to find general conflicts search for the git conflict markers (``<<<<<<<``,
+``=======`` or ``>>>>>>>``).
 
-There are two kinds of conflicts you will encounter in the output that the
-script produces:.
+Fix the conflict manually and stage the changed files::
 
-* Submodule conflict ::
+  git add <FILE_1> <FILE_2> # stages file(s)
 
-    Please manually handle the merging of each conflicted submodule.
-    This can be accomplished with the following steps:
-    - go to submodule (src/tools/cargo), and either merge commit <COMMIT_HASH>
-      or update to an existing commit which has merged those changes
-    - come back to superproject and run:
+Deletion conflict
+"""""""""""""""""
 
-          git add src/tools/cargo
+In order to find deletion conflicts search for ``<<<PULL-UPSTREAM>>>``. Automation
+inserts this marker into the first line of the conflicting file.
 
-      to record the above merge or update
-    - resolve any other conflicts in the superproject
-    - commit the resulting index in the superproject
+This conflict happens when we made changes to a file and it gets deleted by upstream.
+To resolve this, remove the file::
 
-  The message already describes how to fix this conflict. The commands required
-  for this example are::
+  git rm <FILE_1> <FILE_2> # removes file(s) and stages that change
 
-    cd src/tools/cargo
-    git merge <COMMIT_HASH>
-    cd ../../..
-    git add src/tools/cargo
+If this file was a test case used in the traceability matrix, we might need to
+update the traceability matrix, if some section is missing a test case. But finding
+a new testcase should be done in a separate PR.
 
-* General conflicts::
+Commit and push
+^^^^^^^^^^^^^^^
 
-    CONFLICT (content): Merge conflict in src/bootstrap/flags.rs
+After having fixed the conflicts, add a new commit and push it to the PR::
 
-  These are more common conflicts and will require manual fixing depending on
-  the conflict in source.
+  git commit
+  git push
 
-After having fixed the conflicts, finish up the merge by doing the following::
+Review
+^^^^^^
 
-  git add . # add your changes to the staging area
-  git merge --continue # finish the merge
-  git push --set-upstream origin <NEW_PULL_BRANCH> # push out the pull request branch
-
-Finally generate the PR description with::
-
-  ferrocene/tools/pull-upstream/generate_pr_body.py origin <BASE_BRANCH> <NEW_PULL_BRANCH>
-
-This will output a markdown list of the pull requests that have been pulled
-from the upstream Rust repository. Paste that into the PR description of
-the pull request you create from your `<NEW_PULL_BRANCH>` branch.
-You may omit `<NEW_PULL_BRANCH>` in which case the script will use the currently checked
-out branch.
+Now another developer needs to review and sign off the PR (see "Upstram Pull without
+merge conflicts" section above).
```

Comment:
I wonder whether we need instructions like this here? Just saying "commit your changes, and push them for review" would be enough I think?